src/osgAnimation/StatsVisitor.cpp

```diff
@@ -77,10 +77,6 @@ void StatsActionVisitor::apply(ActionStripAnimation& action)
     if (isActive(action))
     {
         _channels.push_back(action.getName());
-        double value;
```

Comment:
Removal looks approrpiate, code looks to have been experimental but not actually used for anything in the end.